packages/clay-management-toolbar/src/ManagementToolbar.tsx

```diff
@@ -16,12 +16,20 @@ interface IProps extends React.HTMLAttributes<HTMLElement> {
 	 * Flag that identifies when to show the active state of the component.
 	 */
 	active?: boolean;
+
+	/**
+	 * Set a maximum width on container-fluid.
+	 */
+	fluidSize?:
+		| React.ComponentProps<typeof ClayLayout.Container>['fluidSize']
+		| false;
```

Comment:
@pat270 I don't think it would be necessary to use `false` here, just inheriting the component type would be enough.